src/KubeOps/Operator/Commands/Generators/RbacGenerator.cs

```diff
@@ -104,6 +105,6 @@ public async Task<int> OnExecuteAsync(CommandLineApplication app)
 
         private static IEnumerable<TAttribute> GetAttributes<TAttribute>(Assembly assembly)
             where TAttribute : Attribute =>
-            assembly.GetTypes().SelectMany(t => t.GetCustomAttributes<TAttribute>(true));
+            assembly.GetTypes().SelectMany(t => t.GetCustomAttributes<TAttribute>(true)); // TODO Call IResourceTypeService
```

Comment:
TODO?

Reply:
Oops, I re-did code in `RbacGenerator.GenerateManagerRbac` but didn't remove that `GetAttributes<TAttribute>` method, though it is no longer being used. Fixed